### PR TITLE
Fixed opening of links in an external app.

### LINKED ISSUE #1980: Hiperlinks are broken
Href attribute is broken and unable to load the requested links. This occurs throughout all of the Configurator Hiperlinks:

hiperlinkswrong

I can see that removing `ref: noopener nooreferrer` it works as expected, but idont know if remove all refs is the right solution:

hiperlinksok

### PR DESCRIPTION
**Pull request by mikeller**
Fixes #1980.

Open the app Window as a NW.js window, which is what NW.js was designed for.
This will re-enable the `new-win-policy` NW.js event handler that is required to open links in an external app.

Also moves the `close` handler into `main.js`, so that `eventPage.js` does not need to be used in NW.js any more.

### CODE REVIEW

**Comment by mikeller** [comment]
@McGiverGim: It was a lot of trial and error. 😅

**Comment by Docteh** [comment]
What about the minimum size of the window? I like the dark task bar on windows 10, its probably respecting dark mode now.

image


image


```json
  "window": {
    "show": false,
    "icon": "images/bf_icon_128.png",
    "id": "main-window",
    "min_height": 550,
    "min_width": 1024
  },
```

**Comment by McGiverGim** [comment]
True, we have a lot of tabs and css working for this minimum sizes. Maybe when merged the Cordova PR it adjust better, but for this release I think is better to have it.

File: package.json [hunk 1]
```diff
@@ -12,8 +11,13 @@
...
   "window": {
-    "show": false,
-    "icon": "images/bf_icon_128.png"
+    "icon": "images/bf_icon_128.png",
+    "id": "main-window",
+    "frame": "chrome",
+    "innerBounds": {
+        "minWidth": 1024,
+        "minHeight": 550
+    }
```
Comment (Docteh) [inline package.json:20]: @McGiverGim This innerBounds section not being effective is what I'm talking about with the min_width/min_height suggestion.

Comment (McGiverGim) [reply package.json:20]: Maybe we must try to open the window from JavaScript and pass them as parameters, similar to the Chrome version.
My PR opened it from JavaScript but I have not passed parameters.

**Comment by mikeller** [comment]
@Docteh: Sorry, my bad. I simply copied the window settings from `eventPage.js` without verifying that they are actually in the correct format for NW. Fixed now - seems to be working fine.

**Comment by TheIsotopes** [comment]
@mikeller i tested this and found a bug
in cli tab the buttons `load from file` and `save to file` does not work.

**Comment by McGiverGim** [comment]
Maybe we are using chrome load/save features and not nw features?

**Comment by McGiverGim** [comment]
I confirm:
https://github.com/betaflight/betaflight-configurator/blob/433fe20ab47648ecbf6b98abef8d5c925614dd90/src/js/tabs/cli.js#L142
 and I suppose we must use: https://nwjs.readthedocs.io/en/nw13/References/File%20Dialogs/

Maybe we must make a wrapper similar to the one of Clipboard.js

**Comment by McGiverGim** [comment]
It seems that since nw2, as we suspected, the window opened from the background page is treated as a background page. This is the error when trying to open a file:
image

So the problem starts here...

**Comment by mikeller** [comment]
From the look of it this goes deeper than what we are doing with the clipboard - without the Chrome filesystem API we'll have to use the node filesystem module, and with this we'll have to move to requiring modules.
I have disabled NW2 for now - the scope of moving to NW2 seems to be too big and risky to do before the upcoming release.

Confirmed working:
- loading links in an external browser;
- opening files.

**Comment by TheIsotopes** [comment]
confirmed, working again 👍

**Comment by mikeller** [comment]
@MaximeAmadasi: You should probably be aware of this, as it changes some of the underlying code that you are working on as well, so you can rebase on this as soon as it has been merged.

**Comment by MaximeAmadasi** [comment]
> True, we have a lot of tabs and css working for this minimum sizes. Maybe when merged the Cordova PR it adjust better, but for this release I think is better to have it.
> 

The cordova PR will make the headerbar a flex container. It's design to work with a miminum window width of 1024px. And when the window width is below 575px, it switchs to the mobile headerbar (which is the same header bar but more compact)

**Pull request merged**